src/Libraries/UnitsUI/ViewModelCustomization.cs

```diff
@@ -0,0 +1,296 @@
+﻿using Dynamo.Controls;
+using Dynamo.Core;
+using Dynamo.Graph.Nodes;
+using Dynamo.UI.Commands;
+using Dynamo.ViewModels;
+using DynamoUnits;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UnitsUI
```

Comment:
think this entire class still needs summaries

Reply:
Done.